sql/resource_sql_endpoint.go

```diff
@@ -95,7 +95,7 @@ func ResourceSqlEndpoint() common.Resource {
 			}
 			var se sql.CreateWarehouseRequest
 			common.DataToStructPointer(d, s, &se)
-			common.SetForceSendFields(&se, d, []string{"enable_serverless_compute", "enable_photon"})
+			common.SetForceSendFields(&se, d, []string{"enable_serverless_compute", "enable_photon", "auto_stop_mins"})
```

Comment:
We need to send it for update as well... it will be better to defined a single constant array and use it for both update & create